example/glue/EnvironmentDelegate.cpp

```diff
@@ -21,9 +21,40 @@
 #include "omrExampleVM.hpp"
 
 #include "AtomicOperations.hpp"
-#include "EnvironmentBase.hpp"
+#include "EnvironmentStandard.hpp"
 #include "EnvironmentDelegate.hpp"
 #include "GCExtensionsBase.hpp"
+#include "SublistFragment.hpp"
+
+OMR_VMThread *
+MM_EnvironmentDelegate::attachVMThread(OMR_VM *omrVM, const char *threadName, uintptr_t reason)
+{
+	OMR_VMThread *omrVMThread = NULL;
+	omr_error_t rc = OMR_ERROR_NONE;
+
```

Comment:
I tend to agree, but there are Java-specific concerns that require delegation. We can discuss this as I think we may be approaching a time where we need to look at moving some of the low-level threading support from Java into OMR..